incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -170,6 +174,22 @@ func (r *HierarchyConfigReconciler) onMissingNamespace(log logr.Logger, nm strin
 	}
 }
 
+// removeExcludedNamespaceLabel removes the `hnc.x-k8s.io/excluded-namespace`
+// label from the namespace.
+func (r *HierarchyConfigReconciler) removeExcludedNamespaceLabel(log logr.Logger, nsInst *corev1.Namespace) {
+	old := nsInst.Labels
+	new := map[string]string{}
```

Comment:
Why not `delete(old, api.LabelExcludedNamespace)`?

Reply:
Ah right updated.